l10n_be_cooperator/models/subscription_request.py

```diff
@@ -8,6 +9,16 @@ class SubscriptionRequest(models.Model):
         [("scrl", "SCRL"), ("asbl", "ASBL"), ("sprl", "SPRL"), ("sa", "SA")]
     )
 
+    def get_accounting_account(self):
+        try:
+            return super().get_accounting_account()
+        except UserError as error:
+            accounts = self.env["account.account"].search([("code", "=", "416000")])
+            if accounts:
+                return accounts[0]
+            else:
+                raise error
+
```

Comment:
I would make things simpler : remove this and solely rely on module configuration.